Source/WebCore/svg/SVGFEFuncAElement.h

```diff
@@ -21,6 +21,7 @@
 #pragma once
 
 #include "SVGComponentTransferFunctionElement.h"
+#include "SVGComponentTransferFunctionElementInlines.h"
```

Comment:
This seems unfortunate. We try to avoid including `Inlines.h` in non-inlines header files.

Reply:
I understand the problem, and I’ve run into this before, where we have to put things into headers because the bindings generator doesn’t understand in what cases it has to include more.

I don’t have an immediate idea for how to solve it, but I don’t think a bindings generator keyword for this would be super-terrible. I personally don’t think we should do something that assumes we need an `Inlines.h` include whenever that file exists.